File: app/src/main/java/com/trimettransit/tracker/util/Constants2.java
package com.trimettransit.tracker.util;

import com.trimettransit.tracker.BuildConfig;

public class Constants2 {

    public static String getTrimetApiKey() {
        return BuildConfig.TRIMET_API_KEY == null ? "" : BuildConfig.TRIMET_API_KEY.trim();
    }

    public static boolean hasTrimetApiKey() {
        return !getTrimetApiKey().isEmpty();
    }
}

File: app/src/main/java/com/trimettransit/tracker/util/Constants2.kt
package com.trimettransit.tracker.util

import com.trimettransit.tracker.BuildConfig

object Constants2 {
    @JvmStatic
    fun getTrimetApiKey(): String {
        return BuildConfig.TRIMET_API_KEY?.trim() ?: ""
    }

    @JvmStatic
    fun hasTrimetApiKey(): Boolean = getTrimetApiKey().isNotEmpty()
}
